tutorials/prometheus-adapter.md

```diff
@@ -1,15 +1,15 @@
 # Getting started with Prometheus and TimescaleDB
 
-If you ever needed a monitoring solution for your infrastructure than you've
+If you've ever needed a monitoring solution for your infrastructure,  then you've
```

Comment:
are there 2 spaces after the comma, or is github doing some weird spacing there?